website/source/guides/stateful-workloads/portworx.html.md

```diff
@@ -0,0 +1,418 @@
+INFO[0000] PX binaries found at /opt/pwx/bin/px-runc    
+INFO[0000] Initializing as version 2.0.2.3-c186a87 (OCI) 
+...
+INFO[0000] Successfully written /etc/systemd/system/portworx.socket 
+INFO[0000] Successfully written /etc/systemd/system/portworx-output.service 
+INFO[0000] Successfully written /etc/systemd/system/portworx.service 
+```
+
+Since we have created new unit files, please run the following command to reload
+the systemd manager configuration:
+
+```
+sudo systemctl daemon-reload
+```
+
+### Step 5: Start Portworx and Check Status
+
+Run the following command to start Portworx:
+
+```
+$ sudo systemctl start portworx
+```
+Verify the service:
+
+```
+$ sudo systemctl status portworx
+● portworx.service - Portworx OCI Container
+   Loaded: loaded (/etc/systemd/system/portworx.service; disabled; vendor preset
+   Active: active (running) since Wed 2019-03-06 15:16:51 UTC; 1h 47min ago
+     Docs: https://docs.portworx.com/runc
+  Process: 28230 ExecStartPre=/bin/sh -c /opt/pwx/bin/runc delete -f portworx ||
+ Main PID: 28238 (runc)
+...
+```
+Wait a few moments (Portworx may still be initializing) and then check the
+status of Portworx using the `pxctl` command. 
+
+```
+$ pxctl status
+```
+
+If everything is working properly, you should see the following output:
+
+```
+Status: PX is operational
+License: Trial (expires in 31 days)
+Node ID: 07113eef-0533-4de8-b1cf-4471c18a7cda
+	IP: 172.31.53.231 
+ 	Local Storage Pool: 1 pool
+	POOL	IO_PRIORITY	RAID_LEVEL	USABLE	USED	STATUS	ZONE	REGION
+	0	LOW		raid0		50 GiB	4.4 GiB	Online	us-east-1c	us-east-1
+	Local Storage Devices: 1 device
+```
+Once all nodes are configured, you should see a cluster summary with the total
+capacity of the storage pool (if you're using the environment provided in this
+guide, the total capacity will be 150 GB since the external block device
+attached to each client nodes has a capacity of 50 GB):
+
+```
+Cluster Summary
+	Cluster ID: my_test_cluster
+	Cluster UUID: 705a1cbd-4d58-4a0e-a970-1e6b28375590
+	Scheduler: none
+	Nodes: 3 node(s) with storage (3 online)
+...
+Global Storage Pool
+	Total Used    	:  13 GiB
+	Total Capacity	:  150 GiB
+```
+
+### Step 6: Create the `mysql.nomad` Job File
+
+We are now ready to deploy a MySQL database that can use Portworx for storage.
+Create a file called `mysql.nomad` and provide it the following contents:
+
+```
+job "mysql-server" {
+  datacenters = ["dc1"]
+  type        = "service"
+
+  group "mysql-server" {
+    count = 1
+
+    restart {
+      attempts = 10
+      interval = "5m"
+      delay    = "25s"
+      mode     = "delay"
+    }
+
+    task "mysql-server" {
+      driver = "docker"
+
+      env = {
+        "MYSQL_ROOT_PASSWORD" = "password"
+      }
+
+      config {
+        image = "hashicorp/mysql-portworx-demo:latest"
+
+        port_map {
+          db = 3306
+        }
+
+        volumes = [
+          "name=mysql,size=10,repl=3/:/var/lib/mysql",
```

Comment:
I would maybe recommend creating the volume externally, rather than getting folks to create volumes in their jobfile - It'll make it easier to migrate to future storage options.

Reply:
Good idea. I took the volume creation out of the job file and added a step in the guide showing users how to create/inspect Portworx volumes.